### DISCUSSION TITLE
게스트 유저의 게스트의 비밀댓글 조작 로직

### DISCUSSION THREAD

**Discussion by zereight** [darass-team/2021-darass#727] [Ideas]
게스트의 비밀댓글이 달려 있고, 현재 게스트인 상황.
"조회" 버튼을 누르고 비밀번호를 입력해서 댓글의 내용을 확인할 수 있음.

여기서,

1. 게스트는 비밀댓글을 조회하여 보이는 상태에서만 수정이 가능한가?
아니면
게스트는 비밀댓글이 안보이더라도 수정버튼을 누르고 비밀번호를 맞추면, 댓글 내용이 보이면서 수정이 가능한가?

2. 게스트는 비밀댓글을 조회하여 보이는 상태에서만 삭제가 가능한가?
아니면
게스트는 비밀댓글이 안보이더라도 삭제버튼 누르고, 비밀번호 맞추면 바로 삭제할 수 있는가?

**Comment by pjy1368** [comment]
티스토리는 비밀 댓글은 수정과 삭제도 안되네. 얘네 좀 못 만들었다고 느끼는게 ㅋㅋㅋㅋㅋㅋ 비밀 댓글은 작성했던 비로그인 유저가 조회/수정/삭제 다 못한다 ㅋㅋㅋㅋㅋ

**Reply by pjy1368** [reply]
일단 난 비밀 댓글이 보여야 수정/삭제가 가능한게 좋은듯해
